src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -1,28 +1,28 @@
-﻿# `ICursor` Notes
+﻿# `DataViewRowCursor` Notes
 
 This document includes some more in depth notes on some expert topics for
-`ICursor` implementations.
+`DataViewRow` and `DataViewRowCursor` derived classes.
 
 ## `Batch`
 
-Some cursorable implementations, like `IDataView`, can through
-`GetRowCursorSet` return a set of parallel cursors that partition the sequence
-of rows as would have normally been returned through a plain old
-`GetRowCursor`, just sharded into multiple cursors. These cursors can be
-accessed across multiple threads to enable parallel evaluation of a data
-pipeline. This is key for the data pipeline performance.
+`IDataView.GetRowCursorSet` can return a set of parallel cursors that
+partition the sequence of rows as would have normally been returned through a
+more typical call to `GetRowCursor`. These cursors can be accessed across
+multiple threads to enable parallel evaluation of a data pipeline. This is key
+for the data pipeline performance during featurization.
 
 However, even though the data pipeline can perform this parallel evaluation,
 at the end of this parallelization we usually ultimately want to recombine the
 separate thread's streams back into a single stream. This is accomplished
```

Comment:
>back into a single stream [](start = 26, length = 25)

might want to say why.

Reply:
OK, I tried to add a bit more detail about this.